EiDiA_frontend/src/components/exportView/subcomponents/EditorTools.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import IconButton from '@material-ui/core/IconButton'
+import {GrTextAlignCenter, GrTextAlignFull, GrTextAlignLeft, GrTextAlignRight} from 'react-icons/gr';
+import {AiOutlineBold, AiOutlineItalic, AiOutlineStrikethrough, AiOutlineUnderline} from 'react-icons/ai';
+import "draft-js/dist/Draft.css";
+
+const styles = {
+    div: {
+        padding: "7px"
+    },
+    iconClicked: {
+        color: "black"
+    }
+}
+
+export default class EditorTools extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+        // If true: mark as active
+            bold: false,
+            italic: false,
+            underlined: false,
+            strikethrough: false,
+            alignments: {
+                left: true,
+                center: false,
+                right: false,
+                justify: false
+            }
+        }
+        this.setInlineStyle = this.setInlineStyle.bind(this);
+        this.setAlignment = this.setAlignment.bind(this);
+    }
+
+    setInlineStyle(e, inlineStyle) {
+        e.preventDefault();
+        this.props.onAction1(inlineStyle);
+        let newState = this.state;
+        newState[inlineStyle] = !newState[inlineStyle];
+        this.setState(newState);
+    }
+
+    setAlignment(e, alignmentStyle) {
+        e.preventDefault()
+        this.props.onAction2(alignmentStyle);
+        let newState = this.state;
+        const newFlag = !newState.alignments[alignmentStyle]
+        for (let key in newState.alignments) {
+            newState.alignments[key] = false
+        }
+        newState.alignments[alignmentStyle] = newFlag;
```

Comment:
```suggestion
        newState.alignments[alignmentStyle] = true;
```

für alignment braucht man eine andere Strategie, sonst haben wir am ende überhaupt kein alignment ausgewählt